hnn_core/viz.py

```diff
@@ -538,6 +533,9 @@ def plot_spikes_raster(cell_response, trial_idx=None, ax=None, show=True):
         cell_type_times, cell_type_ypos = [], []
         for gid in cell_type_gids:
             gid_time = spike_times[spike_gids == gid]
+            _, gid_time = _get_plot_data_trange(
+                gid_time, gid_time, cell_response.times[0],
+                cell_response.times[-1])
```

Comment:
Is there a reason why we need this? I think any spike times outside the x-limits will get cropped when plotting.